frontends/web/src/components/rates/rates.tsx

```diff
@@ -110,13 +112,16 @@ function Conversion({
   skipUnit,
   active,
   noAction,
-  children,
+  sign,
 }: PropsWithChildren<Props>): JSX.Element | null {
 
-  let formattedValue = '---';
+  let unavailableValue = ' ---';
+  let formattedValue = unavailableValue;
```

Comment:
Is the space before the dashes `' ---';` still needed?

Also `unavailableValue` is not used anymore, I think you can just directly do `let formattedValue = '---';`

Reply:
Right! Done :+1: